### PR TITLE
chm-390 National Report

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrate National Report from chm to absch project

### Designs
refer to absch national report and chm.cbd.int
image

### CODE REVIEW

File: app/views/forms/directives/document-date.vue [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<template>
+    <div v-if="(document.createdOn !=undefined || document.updatedOn !=undefined)" >
+        <div class="float-end record-Options-div">
+            <span class="xlabel" ng-if="createdOn">
+                published: <span class="text-primary bold" >{{formatDate(document.createdOn, 'DD MMM YYYY')}}</span>       
+            </span>
```
Comment (tahmeedshah) [inline app/views/forms/directives/document-date.vue:6]: move text to translation file.
also , do we need this file ?
if not please remove it.

File: app/views/forms/view/chm/national-report.vue [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                </div>   
+                
+                <div v-if="document.title">
+                    <label>{{ t("title") }} </label> 
+                    <ng  v-vue-ng:km-value-ml  :value="document.title" :locales="locale" html></ng>   
+                </div>   
+
+                <div v-if="document.reportType">
+                    <label>{{ t("typeOfDocument") }} </label>   
+                    <div class="km-value">
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/national-report.vue:21]: just confirm , reportType is single or multiple.

File: app/views/forms/view/directives/view-relevant-information.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    <label>{{ t("relevantInfo") }} </label>  
+                 </slot> 
+                <ng v-vue-ng:km-value-ml  :value="documentText" :locales="locale" html ></ng>  
+            </div>
+
+            <div v-if="documentLinks">  
+                <slot name="link">
+                    <label>{{ t("relevantWebsites") }} </label> 
+                 </slot> 
+                <div class="km-value" compare-val>
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/views/forms/view/directives/view-relevant-information.vue:17]: after test, find it doesn't work. remove "compare-val"

File: app/views/forms/directives/document-date.vue (hunk 1 above)
Comment (liyuanmontreal) [reply app/views/forms/directives/document-date.vue:6]: remove document-date file

File: app/views/forms/view/directives/view-relevant-information.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (tahmeedshah) [reply app/views/forms/view/directives/view-relevant-information.vue:17]: please remove all 'compare-val' from this file, it is still there i.e <div class="km-value" compare-val="relevantDocs">

**Review by tahmeedshah — APPROVED** [review]

**Comment by liyuanmontreal** [comment]
image
image

**Review by tahmeedshah — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**